automatic/mysql/update.ps1

```diff
@@ -1,48 +1,57 @@
-Import-Module Chocolatey-AU
+﻿Import-Module Chocolatey-AU
 
-$releases = 'https://dev.mysql.com/downloads/mysql/'
+# The mysql-server repo contains a bunch of tags for "mysql-cluster-VERSION"
+# If I use https://api.github.com/repos/mysql/mysql-server/tags then I'll need to paginate to find "mysql-VERSION"
+# I'm using https://api.github.com/repos/mysql/mysql-server//git/matching-refs/tags so that I'll get a complete list without needing to paginate
+# https://stackoverflow.com/a/74561918
+$releases = 'https://api.github.com/repos/mysql/mysql-server/git/matching-refs/tags'
 
 function global:au_SearchReplace {
-    @{
-        'tools\chocolateyInstall.ps1' = @{
-            "(^[$]url\s*=\s*)('.*')"      = "`$1'$($Latest.URL64)'"
-            "(^[$]checksum\s*=\s*)('.*')" = "`$1'$($Latest.Checksum64)'"
-            "(^[$]checksumType\s*=\s*)('.*')" = "`$1'$($Latest.ChecksumType64)'"
-        }
+  @{
+    'tools\chocolateyInstall.ps1' = @{
+      "(^[$]url\s*=\s*)('.*')"          = "`$1'$($Latest.URL64)'"
+      "(^[$]checksum\s*=\s*)('.*')"     = "`$1'$($Latest.Checksum64)'"
+      "(^[$]checksumType\s*=\s*)('.*')" = "`$1'$($Latest.ChecksumType64)'"
     }
- }
+  }
+}
 
 function global:au_GetLatest {
-    $download_page = Invoke-WebRequest -Uri $releases
+  $headers = @{}
+  if (Test-Path Env:\github_api_key) {
+    $headers.Authorization = "token " + $env:github_api_key
+  }
```

Comment:
The GitHub api key should always be required. If not available, this package should not be able to be updated.

Reply:
I changed it to require the env var.

2a49231cfd48471e7545da7c2f7edf4e28d93b0d